practice-one/src/assets/css/text.css

```diff
@@ -9,9 +9,27 @@
 .text-dark {
   color: var(--black-color);
 }
+.text-white {
+  color: var(--white-color);
+}
+
 .navbar-item {
```

Comment:
Why do we have this class in this file?

Reply:
Because in my text.css file, I will define classes for common text styles that will be used across multiple sections. So, I want to use this class for text with a white color, and I can apply this class multiple times.

Reply:
With a common text style, you shouldn't be named with a special name like that. If it is styled for a navbar item, should place it in navbar.css